# Use this file to specify parameters, that do not have default values.
# YAML Files can also be helpfull if you want to save parameters for different use-cases.
# In any way: Default values should go into the .mrtcfg file!
#
# Use 'key: value' pairs, e.g.
# string: 'foo'
# integer: 1234
# float: 1234.5
# boolean: true
# vector: [1.0, 2.0, 3.4]
# map: {"a": "b", "c": "d"}

wheel_base: 0.51
tf_refresh_rate: 60.0
map_frame: 'world'
vehicle_frame: 'vehicle_rear_axle'
initial_x: 0.0 # [m]
initial_y: 0.0 # [m]
initial_yaw: 0.0 #[rad]
